### PR TITLE
Installing terrafom using snap warning

### PR DESCRIPTION
**Pull request by ethervoid**
#### Summary

I've been having permission problems using terraform installed via snap so I decided to include a mention about that and the terraform version needed in the docs

### CODE REVIEW

File: docs/terraform.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ go get github.com/mattermost/mattermost-load-test/cmd/loadtest
 go get github.com/mattermost/mattermost-load-test/cmd/ltparse
 ```
 
-Then install [terraform](https://www.terraform.io/intro/getting-started/install.html), and optionally, install the [AWS CLI tool](https://aws.amazon.com/cli/).
+You're going to need [terraform](https://www.terraform.io/intro/getting-started/install.html) version 0.11. If you're using ubuntu, we don't recommend to install it using Snap because you could have some permission problems.
```
Comment (lieut-data) [inline docs/terraform.md]: This seems a little vague and "unactionable": is there a concrete issue we could link to in the upstream repository that describes the permission issues?

**Review by lieut-data — CHANGES_REQUESTED** [review]
Thanks, @ethervoid -- comment below.

File: docs/terraform.md (hunk 1 above)
Comment (ethervoid) [reply docs/terraform.md]: We don't have issues for it yet. I can create one to add the link here but its something related to ubuntu rather to mattermost. I had the same problems with Docker so I didn't know how to include something specific :\

Comment (ethervoid) [reply docs/terraform.md]: I left only the version addition

Comment (lieut-data) [reply docs/terraform.md]: Understood. I think my worry is that this doesn't really /help/ a community member as it lacks sufficient detail. How will a community member know that their permission error is the same as the one we observed? Or when the issue is resolved, but we forget to update this mention.

3/5, but I'm wondering if we just skip documenting this altogether and start with filing the upstream issue to get the problem resolved instead.

Comment (ethervoid) [reply docs/terraform.md]: I see your point. I'm going to fill a ticket about this issue with snap so we can fix or at least take a look into it.

What do you think to leave this PR just to include the mandatory version for terraform (which is 0.11)?

**Comment by ethervoid** [comment]
[I've seen](https://mattermost.atlassian.net/browse/MM-16988) that is supporting terraform 0.12 as well so closing

**Pull request closed without merging**